packages/api/src/database/queries/entreprises.ts

```diff
@@ -196,9 +196,7 @@ const titreDemandeEntreprisesGet = async (
   }
 
   if (isAdministrationAdmin(user) || isAdministrationEditeur(user)) {
-    const titresCreation = await titresCreationQuery(
-      user.administrationId
-    ).first()
+    const titresCreation = isGestionnaire(user.administrationId)
 
     if (!titresCreation) return []
```

Comment:
Quelque chose comme ça ? la const titresCreation est mal nommée non ?
```suggestion
    if (!isGestionnaire(user.administrationId)) return []
```